test/makerV2Manager_invest.js

```diff
@@ -21,22 +24,33 @@ describe("MakerV2 DSR", function () {
   let makerV2;
   let sai;
   let dai;
+  let jug;
+  let vat;
+  let gov;
 
   before(async () => {
     const m = await deployMaker(deployer, infrastructure);
-    [sai, dai] = [m.sai, m.dai];
+    [sai, dai, jug, vat, gov] = [m.sai, m.dai, m.jug, m.vat, m.gov];
```

Comment:
`jug`, `vat and `gov` are not used anywhere outside this block so they could instead be destructured on the next line along with `migration` and `pot`.